oemof/tools/datapackage.py

```diff
@@ -64,6 +64,15 @@ def read_facade(facade, facades, create, typemap, data, objects,
                 resources):
     """ Parse the resource `r` as a facade.
     """
+    # TODO: Make sure the class var 'required' exists
+    if hasattr(typemap[facade['type']], 'required'):
+        for r in typemap[facade['type']].required:
+            if facade[r] is None:
+                raise AttributeError(
+                    ("Missing attribute `{}` for facade of type `{}` with" +
+                     " name/label `{}`.").\
+                        format(str(r), facade['type'], facade['name']))
+
```

Comment:
Hey. Unless I'm missing something here, there's a better way to do this. See my comments in c034689.